gui/webdisplay/src/RWebWindowsManager.cxx

```diff
@@ -170,6 +170,18 @@ void RWebWindowsManager::SetUseConnectionKey(bool on)
    gEnv->SetValue("WebGui.OnetimeKey", on ? "yes" : "no");
 }
 
+//////////////////////////////////////////////////////////////////////////////////////////
+/// Enable or disable single connection mode (default on)
+/// If enabled, the only connection with any web widget is possible
```

Comment:
"one connection only with any web widet.." ?

Reply:
Per widget only one connection will be possible.